diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/AuthInAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/AuthInAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/AuthInAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/AuthInAction.pm	2008-11-10 15:49:23.000000000 +0100
@@ -1,4 +1,4 @@
-package AuthInAction;
+package Athena::AuthInAction;
 
 use strict;
 use CGI;
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/BalanceAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/BalanceAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/BalanceAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/BalanceAction.pm	2008-11-10 15:49:31.000000000 +0100
@@ -1,4 +1,4 @@
-package BalanceAction;
+package Athena::BalanceAction;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/BalanceView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/BalanceView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/BalanceView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/BalanceView.pm	2008-11-10 15:49:38.000000000 +0100
@@ -1,4 +1,4 @@
-package BalanceView;
+package Athena::BalanceView;
 
 use strict;
 use CGI qw(:standard);
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/ChooseLbAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ChooseLbAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/ChooseLbAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ChooseLbAction.pm	2008-11-10 15:49:50.000000000 +0100
@@ -1,4 +1,4 @@
-package ChooseLbAction;
+package Athena::ChooseLbAction;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/ChooseLbView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ChooseLbView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/ChooseLbView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ChooseLbView.pm	2008-11-10 15:49:58.000000000 +0100
@@ -1,4 +1,4 @@
-package ChooseLbView;
+package Athena::ChooseLbView;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/ChooseMethodAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ChooseMethodAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/ChooseMethodAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ChooseMethodAction.pm	2008-11-10 15:50:05.000000000 +0100
@@ -1,4 +1,4 @@
-package ChooseMethodAction;
+package Athena::ChooseMethodAction;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/ChooseMethodView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ChooseMethodView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/ChooseMethodView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ChooseMethodView.pm	2008-11-10 15:50:12.000000000 +0100
@@ -1,4 +1,4 @@
-package ChooseMethodView;
+package Athena::ChooseMethodView;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/Connection.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/Connection.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/Connection.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/Connection.pm	2008-11-10 15:50:39.000000000 +0100
@@ -1,4 +1,4 @@
-package Connection;
+package Athena::Connection;
 
 use strict;
 use IO::Socket::INET;
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/EngineView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/EngineView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/EngineView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/EngineView.pm	2008-11-10 15:50:44.000000000 +0100
@@ -1,4 +1,4 @@
-package EngineView;
+package Athena::EngineView;
 
 use strict;
 use CGI qw(:standard);
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/ErrorView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ErrorView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/ErrorView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ErrorView.pm	2008-11-10 15:50:48.000000000 +0100
@@ -1,4 +1,4 @@
-package ErrorView;
+package Athena::ErrorView;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/FarmView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/FarmView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/FarmView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/FarmView.pm	2008-11-10 15:50:52.000000000 +0100
@@ -1,4 +1,4 @@
-package FarmView;
+package Athena::FarmView;
 
 use strict;
 use CGI qw(:standard);
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/NavFooterView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/NavFooterView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/NavFooterView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/NavFooterView.pm	2008-11-10 15:50:56.000000000 +0100
@@ -1,4 +1,4 @@
-package NavFooterView;
+package Athena::NavFooterView;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/NavHeaderView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/NavHeaderView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/NavHeaderView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/NavHeaderView.pm	2008-11-10 15:51:00.000000000 +0100
@@ -1,4 +1,4 @@
-package NavHeaderView;
+package Athena::NavHeaderView;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/PhysView.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/PhysView.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/PhysView.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/PhysView.pm	2008-11-10 15:51:04.000000000 +0100
@@ -1,4 +1,4 @@
-package PhysView;
+package Athena::PhysView;
 
 use strict;
 use CGI qw(:standard);
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/UpdateFarmAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/UpdateFarmAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/UpdateFarmAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/UpdateFarmAction.pm	2008-11-10 15:51:08.000000000 +0100
@@ -1,4 +1,4 @@
-package UpdateFarmAction;
+package Athena::UpdateFarmAction;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/UpdatePhysAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/UpdatePhysAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/UpdatePhysAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/UpdatePhysAction.pm	2008-11-10 15:51:12.000000000 +0100
@@ -1,4 +1,4 @@
-package UpdatePhysAction;
+package Athena::UpdatePhysAction;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/Request.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/Request.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/Request.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/Request.pm	2008-11-10 15:51:57.000000000 +0100
@@ -1,4 +1,4 @@
-package Request;
+package Athena::WebAPI::Request;
 
 use strict;
 use CGI qw(:standard);
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/Response.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/Response.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/Response.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/Response.pm	2008-11-10 15:52:01.000000000 +0100
@@ -1,4 +1,4 @@
-package Response;
+package Athena::WebAPI::Response;
 
 use strict;
 use CGI qw(:all);
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/stub.Form.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/stub.Form.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/stub.Form.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/stub.Form.pm	2008-11-10 15:52:14.000000000 +0100
@@ -1,4 +1,4 @@
-package Form;
+package Athena::WebAPI::Form;
 
 use strict;
 use CGI qw(:standard);
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/stub.HTML.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/stub.HTML.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/stub.HTML.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/stub.HTML.pm	2008-11-10 15:52:17.000000000 +0100
@@ -1,4 +1,4 @@
-package HTML;
+package Athena::WebAPI::HTML;
 
 use strict;
 use @@PACKAGE@@::WebAPI::Util;
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/stub.Table.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/stub.Table.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/stub.Table.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/stub.Table.pm	2008-11-10 15:52:21.000000000 +0100
@@ -1,4 +1,4 @@
-package Table;
+package Athena::WebAPI::Table;
 
 use strict;
 use @@PACKAGE@@::WebAPI::Util;
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/Style.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/Style.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/Style.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/Style.pm	2008-11-10 15:52:05.000000000 +0100
@@ -1,4 +1,4 @@
-package Style;
+package Athena::WebAPI::Style;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/Util.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/Util.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/Util.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/Util.pm	2008-11-10 15:52:08.000000000 +0100
@@ -1,4 +1,4 @@
-package Util;
+package Athena::WebAPI::Util;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/View.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/View.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/WebAPI/View.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/WebAPI/View.pm	2008-11-10 15:52:11.000000000 +0100
@@ -1,4 +1,4 @@
-package View;
+package Athena::WebAPI::View;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/ViewEngineAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ViewEngineAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/ViewEngineAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ViewEngineAction.pm	2008-11-10 15:51:16.000000000 +0100
@@ -1,4 +1,4 @@
-package ViewEngineAction;
+package Athena::ViewEngineAction;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/ViewFarmAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ViewFarmAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/ViewFarmAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ViewFarmAction.pm	2008-11-10 15:51:20.000000000 +0100
@@ -1,4 +1,4 @@
-package ViewFarmAction;
+package Athena::ViewFarmAction;
 
 use strict;
 
diff -Naur ath-2.2.0/src/perl/package/Athena/lib/Athena/ViewPhysAction.pm ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ViewPhysAction.pm
--- ath-2.2.0/src/perl/package/Athena/lib/Athena/ViewPhysAction.pm	2008-01-08 20:54:20.000000000 +0100
+++ ath-2.2.0.oden/src/perl/package/Athena/lib/Athena/ViewPhysAction.pm	2008-11-10 15:51:24.000000000 +0100
@@ -1,4 +1,4 @@
-package ViewPhysAction;
+package Athena::ViewPhysAction;
 
 use strict;
 
